content/en/integrations/guide/data-center-migration.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Datadog Data Center Migration
+kind: guide
+description: Learn how to migrate from one Datadog instance to another instance.
+further_reading:
+- link: "https://github.com/DataDog/datadog-sync-cli"
+  tag: "GitHub"
+  text: "datadog-sync-cli"
+- link: "/agent/configuration/dual-shipping/"
+  tag: "Documentation"
+  text: "Learn about Dual Shipping"
+- link: "/integrations/guide/azure-portal/"
+  tag: "Documentation"
+  text: "Managing the Azure Native integration"
+---
+
+## Overview
+
+If you are interested in migrating Datadog data centers, follow these instructions to plan your migration:
+
+1. Create a Datadog account.
+1. Migrate Datadog [resources or assets-pick one and go with it] using the `datadog-sync-cli` package and/or [Dual Shipping][4].
```

Comment:
```suggestion
1. Migrate Datadog [resources or assets-pick one and go with it] using the `datadog-sync-cli` package and/or [Dual Shipping][4].
```
- Missing link here.
- Rephrase "resources or assets-pick one and go with it" - too casual
- "and/or Dual Shipping": Is Dual Shipping optional? It's not clear from the instructions below. If so, say something like "Optionally, use Dual Shipping to prevent data loss."